Automattic/jetpack
# Make payments-intro pattern filtering more robust

Fixes paYJgx-2tT-p2

#### Changes proposed in this Pull Request:

* Makes the payments-intro block pattern filtering more robust - prevents the block crashing when used with blocks without categories

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






1. Ensure you don't have coblocks installed 
2. Open Gutenberg 
3. Insert a Payments block, notice it doesn't crash and the pattern selector still works
4. Install coblocks (free plugin if not pre-installed)
5. Open Gutenberg
6. Insert a Payments block, notice it does crash
7. Use this Jetpack branch
8.  Notice the crash has stopped and the pattern selector still works
